GMnetENGINE.gmx/scripts/htme_config.gml

```diff
@@ -41,6 +49,11 @@ self.udphp_master_ip = "95.85.63.183";
  */
 self.udphp_master_port = 6510;
 
+// Send port to client on online server
+// You can manually port forward the server port to enable clients to connect
+// Note that this will use data8 to provide the port so dont set it yourself
+self.udphp_provide_server_port=true;

```

Comment:
I don't really get why the entire udphp_provide_server_port logic is needed.
GMnet PUNCH automatically falls back to connecting directly with the server what is actually the problem here I think is that the lobby doesn't do this yet.

So instead of having this to be an option, this should just be added to GMnet without an option to turn it on or off.